Makefile

```diff
@@ -15,6 +15,8 @@ JAEGER_VERSION ?= "$(shell grep -v '\#' jaeger.version)"
 OPERATOR_VERSION ?= "$(shell git describe --tags)"
 STORAGE_NAMESPACE ?= "${shell kubectl get sa default -o jsonpath='{.metadata.namespace}' || oc project -q}"
 KAFKA_NAMESPACE ?= "kafka"
+KAFKA_EXAMPLE ?= "https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/release-0.12.x/examples/kafka/kafka-ephemeral.yaml"
+KAFKA_YAML ?= "https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.12.1/strimzi-cluster-operator-0.12.1.yaml"
```

Comment:
I have raised this in the previous PR. Consider defining just the kafka version as a variable.

Reply:
@pavolloffay I noted your previous comment, but did not have a chance to respond.  Please note my other response to @jpkrohling above.  The purpose of this change is not just to permit testing different versions from Strimzi, but to enable testing from AMQ streams, where CRs would come from an entirely different repo.